packages/react-composites/src/composites/CallComposite/adapter/CallAdapter.ts

```diff
@@ -396,6 +396,29 @@ export interface CallAdapterCallOperations {
    * @public
    */
   stopScreenShare(): Promise<void>;
+  /* @conditional-compile-remove(raise-hand) */
+  /**
+   * Raise hand for current user
+   *
+   * @public
+   */
+  raiseHand(): Promise<void>;
+  /* @conditional-compile-remove(raise-hand) */
+  /**
+   * lower hand for current user
+   *
+   * @public
+   */
+  lowerHand(): Promise<void>;
+  /* @conditional-compile-remove(raise-hand) */
+  /**
+   * Lower hand for participants from the call.
+   *
+   * @param userId - List of Ids of the participant
+   *
+   * @public
+   */
+  lowerHands(userIds: string[]): Promise<void>;
```

Comment:
I am curious would there ever be an option where you can lower 3 peoples but not the other 4? I question if it needs these parameters and if it should just be an all or nothing?